_resources/port1.0/group/deprecated-1.0.tcl

```diff
@@ -45,6 +46,7 @@ proc deprecated.deprecate_port {} {
         depends_lib
         depends_run
         archive_sites
+        known_fail yes
```

Comment:
We can do this for now, but be aware that setting known_fail conditional on the Xcode version (which is not considered when building portindexes) could be a problem in the future. See what Josh said in the comments of f8697ede6cf0d2738e30154a065bb5d7c873efbc.